README.md

```diff
@@ -19,3 +19,22 @@ API request that resulted in a `401`.
 If your application does not wish to log these actions at all, use
 `\Psr\Log\NullLogger` for any constructors that require a
 `\Psr\Log\LoggerInterface`.
+
+
+## Overview of main classes
+
+### Client
+MPX API implementation of Guzzle ClientInterface. As a Client it doesn’t do anything extra but suppress errors to force a returning HTTP 200.
+It also handles XML from responses
+
+### UserSession
+A wrapper around Client, implementing all the stuff from Guzzle ClientInterface, and adding sign and singout functionalities.
```

Comment:
Let's reverse the order of this - the important part is the session, not the method implementations.

> Manages authenticated sessions and proxies API calls with a ClientInterface implementation, automatically refreshing expired API tokens as required.